tasks/73_check-for-palindromes/check-for-palindromes.js

```diff
@@ -0,0 +1,5 @@
+export const palindrome = (str) => {
+  const nonAlphanumeric = str.replace(/[_\W]/g, '').toLowerCase();
+  const result = nonAlphanumeric.split('').reverse().join('').toLowerCase();
+  return nonAlphanumeric === result;
```

Comment:
follow up: how would you solve this task without split().reverse().join()?

Reply:
`export const palindrome = (str) => {
  const nonAlphanumeric = str.replace(/[_\W]/g, '').toLowerCase();
  return nonAlphanumeric.split('').every((char, i) => {
    return char === nonAlphanumeric[nonAlphanumeric.length - i - 1];
  });
};`

It should work, master.